migrate/docker/migrate.sh

```diff
@@ -165,7 +168,7 @@ sed -i "$start s/.*/listen_addresses = '*'/" $PG_CFG_PATH
 echo "Patching Postgresql HBA file"
 echo "host all all all md5" >> db/pg_hba.conf
 
-chown -R www-data:docker *
+chown -R www-data:docker ./*
```

Comment:
Please use the full path. This has potential to fail.

Reply:
Thanks for the feedback, will change asap.